matlab/test/arrow/type/tTimestampType.m

```diff
@@ -23,6 +23,10 @@
         ClassName = "arrow.type.TimestampType"
     end
 
+    properties(TestParameter)
+        TimeZone={'America/Anchorage', ''}
```

Comment:
Could you add a comment here like the following?

"Test against both "Zoned" (i.e. non-empty TimeZone value) and "Unzoned" (i.e. empty TimeZone value)."